conformance/utils/tester/tester.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tester
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+// Tester allows users to add custom logging shims to conformance tests.
+type Tester interface {
+	testing.TB
+
+	// Run calls a subtest and wraps the testing.T.Run call
+	Run(name string, f func(Tester))
+
+	// Parallel should pass through to testing.T.Parallel
+	Parallel()
+}
+
+// New returns a new impl.
+func New(t *testing.T) Tester {
+	return &impl{T: t}
+}
+
+// impl performs basic wrapping of testing.T
+type impl struct {
+	*testing.T
```

Comment:
nit: The concrete impl could be a `Tester` as well, then this could just be "LoggingTester" or something

Reply:
renamed to `loggingTester`